kong/templates/nginx.lua

```diff
@@ -1,11 +1,11 @@
 return [[
 pid pids/nginx.pid;
-error_log ${{PROXY_ERROR_LOG}} ${{LOG_LEVEL}};
-
-> if wasm and wasm_dynamic_module then
+> if wasm then
```

Comment:
I think the `wasm_dynamic_module` test should be kept. If nginx is built with static wasmx module, `wasm_dynamic_module` will be empty and this will cause an error.